src/swap/SwapScreen.tsx

```diff
@@ -55,13 +58,43 @@ const DEFAULT_SWAP_AMOUNT: SwapAmount = {
 
 type Props = NativeStackScreenProps<StackParamList, Screens.SwapScreenWithBack>
 
+function getNetworkFee(quote: QuoteResult | null, networkId?: NetworkId) {
+  if (quote && quote.preparedTransactions?.type === 'possible') {
+    return {
+      networkFee: quote.preparedTransactions.maxGasFeeInDecimal,
+      feeTokenId: quote.preparedTransactions.feeCurrency.tokenId,
+    }
+  }
```

Comment:
We could use `getFeeCurrencyAndAmount` here, it would allow displaying a fee when for the `need-decrease-spend-amount-for-gas` type too.
https://github.com/valora-inc/wallet/blob/41eae0760b73f3ac7b5072907bf852813e7ba3e8/src/viem/prepareTransactions.ts#L368

Reply:
updated!